docs/README.md

```diff
@@ -70,41 +77,85 @@ root = "./app"
 
 ### Third-party modules
 
-Third-party modules are external Python packages that are not part of the standard library or your own project’s code. These are typically installed using a package manager like uv or pip and include libraries such as `requests`, `numpy`, or `django`.
+Third-party modules are Python packages that are not part of your project or the standard library.
+These are usually dependencies of your project, and are installed using a package manager like uv or
+pip. Common examples include libraries such as `requests`, `numpy`, or `django`.
```

Comment:
```suggestion
These are usually declared as project dependencies in a `pyproject.toml` file or similar,
and are installed using a package manager like uv or pip. Examples of popular third-party
libraries are `requests`, `numpy` and `django`.
```

Reply:
I wasn't really sure what "these are usually dependencies of your project" really meant in your current text. It feels almost tautological -- third-party modules are dependencies, yes, but what's the definition of a dependency? It's a third-party module.

Reply:
Haha see, I think the point is to provide that sort of tautological "third-party modules is just a fancy word for dependencies" statement since we're explaining to a beginner what we mean by this concept. I expect people to know what a "dependency" is. I can play with this language though...

Reply:
> I expect people to know what a "dependency" is

Ah, I do not!! I've seen lots of Python noobs who've been forced to type checkers by the company they're working for, and who know very little at all about Python packaging 😄

Reply:
```suggestion
These are usually declared as project dependencies in a `pyproject.toml` or `requirements.txt` file
and installed using a package manager like uv or pip. Examples of popular third-party
libraries are `requests`, `numpy` and `django`.
```

Reply:
That's sort of not the target audience for _this_ documentation, but anyway I'm fine with changing it.

Reply:
that LGTM!